clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy

```diff
@@ -120,12 +121,21 @@ class Keys implements KeyParser {
     result
   }
 
+  static String encode(String value) {
+    return value.replace([':' : '%3A', '%' : '%25'])
+  }
+
+  static String decode(String value) {
+    return value.replace(['%3A' : ':', '%25' : '%'])
+  }
```

Comment:
Since this is a backwards-incompatible change, it needs to be put behind a feature flag that is disabled by default.

Reply:
Good point. Though, while I'm looking at this, I'm wondering why the group name is even part of the key? The security group ID and the VPC ID should be enough to ensure uniqueness.